src/main/java/com/google/sps/servlets/AddOrganizationServlet.java

```diff
@@ -39,8 +45,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String hourOpen = request.getParameter("hour-open");
     String hourClosed = request.getParameter("hour-closed");
 
-    //TODO: get timestamp with transactions instead
-    long timestampMillis = System.currentTimeMillis();
+    long secondSinceEpoch = Instant.now().toEpochMilli();
```

Comment:
seconds since epoch or milliseconds. Could you also add a comment here since I dont understand what this does